designer/client/src/PageEdit.tsx

```diff
@@ -480,6 +518,23 @@ export class PageEdit extends Component<Props, State> {
             </>
           )}
 
+          {controller === ControllerType.Repeat && (
+            <Input
+              id="page-repeat-title"
+              name="repeat-name"
```

Comment:
Minor comment, just for consistency sake (to match up with PageCreate)

```suggestion
              name="repeat-title"
```